cli/authenticating-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,27 @@
+[id="authenticating-by-using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Authenticating by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: roxctl-cli-authentication
+
+toc::[]
+
+[role="_abstract"]
+For authentication, you can use an API token, your administrator password, or the `roxctl login` command.
+
+The following are some guidelines for the effective use of API tokens:
+
+* Use an API token in a production environment with continuous integration (CI). Each token is assigned specific access permissions, providing control over the actions it can perform. In addition, API tokens do not require interactive processes, such as browser-based logins, making them ideal for automated processes. These tokens have a time-to-live (TTL) of 1 year, providing a longer validity period for seamless integration and operational efficiency.
+
+* Use your administrator password only for testing purposes. Do not use it in the production environment.
```

Comment:
I'd word it that it's not recommended, not sure how the proper naming is.

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#recommend
As per the guidelines we do not use "recommend" in our documentation. 
@kcarmichael08 Please correct me if I'm wrong

Reply:
This is correct - we try to avoid recommend if possible per Style guidelines: https://redhat-documentation.github.io/supplementary-style-guide/#recommend